src/store.js

```diff
@@ -472,7 +472,7 @@ function EvolvStore(options) {
         })
         const pruned = objects.prune(effectiveGenome, active);
         Object.keys(pruned).forEach(function(key) {
-          activeVariants.add(key.concat(':', strings.hashCode(JSON.stringify(pruned[key]))));
+          activeVariants.add(key.concat(':', strings.hashCode(pruned[key])));
```

Comment:
Why not use the import directly here rather than exporting it out through the _strings.js_ file and then also remove the export from that file completely?
```typescript
import { objectHash } from '@sentienttechnologies/hashing';
```

```suggestion
          activeVariants.add(key.concat(':', objectHash(pruned[key])));
```

Reply:
strings is already imported and used in multiple places. I think it's good to keep this pattern as changing it will cause many changes. Al the `strings.[method]`s should change respectfully

Reply:
I'm not saying the other usages of `strings` need to be refactored. I'm just saying that it is unnecessary (and unclear) to re-export/alias `objectHash()` as `hashCode()`. This line is the only usage of the former `strings.hashCode()`.

Reply:
In fact, the `@sentienttechnologies/hashing` module itself exports a function named `hashCode()`.